webui/module/Client/src/Client/Model/ClientModel.php

```diff
@@ -73,16 +86,15 @@ public function getDotClients(&$bsock = null)
      *
      * @return array
      */
-    public function getClient(&$bsock = null, $client = null)
+    public function getClient(&$bsock = null, $client)
     {
-        if (isset($bsock, $client)) {
-            $cmd = 'llist client="' . $client . '"';
-            $result = $bsock->send_command($cmd, 2);
-            $client = \Zend\Json\Json::decode($result, \Zend\Json\Json::TYPE_ARRAY);
-            return $client['result']['clients'];
-        } else {
+        if (!isset($bsock)) {
```

Comment:
I dont quite get this change.  I feel like calling this function with `$client == null` should still throw an exception as the command is wrong.

Reply:
I guess the idea is that the caller checks this and if he doesnt then `call_json` will throw the exception ?
Im fine with that